homeassistant/components/fritz/binary_sensor.py

```diff
@@ -68,7 +68,7 @@ async def async_setup_entry(
         if description.is_suitable(connection_info)
     ]
 
-    async_add_entities(entities, True)
+    async_add_entities(entities)
```

Comment:
We don't need to set `update_before_add` to `True` because we do an initial config entry refresh during setup, before adding platforms